app/views/application/_index_actions.html.slim

```diff
@@ -2,7 +2,7 @@ td= link_to t_action(:show), subject
 td= link_to t_action(:print), { controller: controller_name, action: :show, id: subject.id, print: 'true' }, target: '_blank'
 - if ['clients', 'volunteers'].include? controller_name
   td= link_to t_model(Journal), { controller: 'journals', action: 'index', "#{controller_name.singularize}_id": subject.id }
-td= link_to t_action(:edit), { controller: controller_name, action: :edit, id: subject.id }
-td
-  - if policy(subject.class).destroy?
-    = link_to t_action(:destroy), subject, confirm_deleting(subject)
+- if policy(subject.class).index?
+  td= link_to t_action(:edit), { controller: controller_name, action: :edit, id: subject.id }
```

Comment:
If I remove the controller: controller_name then when I visit a volunteer's show and try to edit their assignment, it renders the volunteer edit.